demos/tests/cases.py

```diff
@@ -222,6 +222,22 @@ def device_cases(*args):
             }),
         ],
     )),
+    
+    PythonDemo(name='text_detection_demo', test_cases=combine_cases(
```

Comment:
You need to update the name of the demo here. Plus, the `name` parameter of `PythonDemo` has by now been replaced by `subdirectory`, so you also need to update that (you might want to rebase to current `develop`).

Reply:
I took `cases.py` from original `develop` branch.

> you might want to rebase to current `develop`
It seems that I should. But I am not sure how to it to not ruin this pull request.

Reply:
> But I am not sure how to do it to not ruin this pull request.
I think, I did